src/index.ts

```diff
@@ -37,17 +37,18 @@ function init(opts:ISimpleSearchBarOptions):void {
 
     var inst:SimpleSearchBar = new SimpleSearchBar(opts);
     instances.push( inst );
-    
-    
+
+
     var $el:HTMLElement = opts.el
-      , $inp:HTMLInputElement = getInp($el)
-      , $btnCls:HTMLButtonElement = <HTMLButtonElement>$el.querySelector(PRF + "btnclose")
-      , justCleared:boolean = false
-      , ignoreBlur:boolean = false
-      , $form:HTMLFormElement = <HTMLFormElement>$el.querySelector(PRF+"form");
-    
+        , $inp:HTMLInputElement = getInp($el)
+        , $btnCls:HTMLButtonElement = <HTMLButtonElement>$el.querySelector(PRF + "btnclose")
+        , $btnSubmit:HTMLButtonElement = <HTMLButtonElement>$el.querySelector(PRF + "btnsubmit")
+        , justCleared:boolean = false
+        , ignoreBlur:boolean = false
+        , $form:HTMLFormElement = <HTMLFormElement>$el.querySelector(PRF+"form");
```

Comment:
Could you just change this so it's an "HTMLElement", rather than an "HTMLFormElement"? 
Then create a 2nd example in the `src/example/` directory, which omits the `.smpSch-form` element, so we can test this working on HTML that has no nested forms in it (as will be the case with a Web Forms site).